go/libkb/util_windows.go

```diff
@@ -183,6 +238,10 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	oldConfig := filepath.Join(oldHome, configName)
 	if oldExists, _ := FileExists(oldConfig); oldExists {
 		if currentExists, _ := FileExists(currentConfig); !currentExists {
+			g.Log.Info("RemoteSettingsRepairman moving from %s to %s", oldHome, currentHome)
```

Comment:
Since some users could be halfway moved, I think we should check for any secretkeys mpack files in oldHome, or anything that looks like the secret store file and run the repairman if they exist.